packages/spektate/src/repository/IAzureDevOpsRepo.test.ts

```diff
@@ -33,13 +35,18 @@ beforeAll(() => {
       "utf-8"
     )
   );
+  prRawResponse = JSON.parse(
+    fs.readFileSync(mockDirectory + "azdo-pr-response.json", "utf-8")
+  );
```

Comment:
should be
```
const  prRawResponse = JSON.parse(
    fs.readFileSync(path(mockDirectory, "azdo-pr-response.json"), "utf-8")
  );
```
and remove `let prRawResponse = {};` at line 17

Reply:
since this JSON.parse is happening in `beforeAll`, i can't get rid of `let prRawResponse = {};`?

Reply:
got it. thanks